sdcm/cluster_k8s/eks.py

```diff
@@ -253,8 +255,12 @@ def allowed_labels_on_scylla_node(self) -> list:
             ('k8s-app', 'aws-node'),
             ('app', 'local-volume-provisioner'),
             ('k8s-app', 'kube-proxy'),
-            ('scylla/cluster', self.k8s_scylla_cluster_name),
         ]
+        if self.tenants_number > 1:
+            allowed_labels_on_scylla_node.append(('app', 'scylla'))
```

Comment:
can you explain those labels ? why it's different between multi tenant setup to regular one ?

Reply:
In case of regular setup we don't want to get manager's Scylla be scheduled onto the Scylla nodes. In case of multi-tenant scenario we intentionally allow more than 1 cluster.